alternator/auth.cc

```diff
@@ -34,7 +34,7 @@ future<std::string> get_key_from_roles(service::storage_proxy& proxy, auth::serv
     const column_definition* salted_hash_col = schema->get_column_definition(bytes("salted_hash"));
     const column_definition* can_login_col = schema->get_column_definition(bytes("can_login"));
     if (!salted_hash_col || !can_login_col) {
-        co_await coroutine::return_exception(api_error::unrecognized_client(format("Credentials cannot be fetched for: {}", username)));
+        co_await coroutine::return_exception(api_error::unrecognized_client(fmt::format("Credentials cannot be fetched for: {}", username)));
```

Comment:
`format` is not a type, it is a function. so we cannot use 
```c++
using format = fmt::format;
```
to define an alias for it. what we can though is to use
```c++
using fmt::format;
```
to expose it in the current namespace. but this does not work, because we already have
```c++
using namespace seastar;
```
which puts all the names in `seastar` in the global namespace. so we already have the equivalent declaration in scylla.

> But on the other hand, maybe it's better to always use namespaces - like we do for "std" - so your longer patch has merit.

i don't intend / dare to change the fact of `using namespace seastar`. it's unusual in an application. but seems we do this on purpose. and it's not in the scope of this change. i just wanted fix the build with the latest seastar, ASAP. to reduce the surprise when some of us want to bump the seastar submodule in (near) future. that'd be frustrating facing ton of errors, when he/she just want to bring a fix in seastar to scylla.

Reply:
> i don't intend / dare to change the fact of using namespace seastar. it's unusual in an application. but seems we do this on purpose. 

Seastar/ScyllaDB were born as one project (then named Urchin). Later Seastar was extracted from it. b958a443044c7182cc5a92ea49d78bdaa337ea7e is shared with the seastar repo. Only later did we add the seastar namespace.

Reply:
btw, we could use fmt::format() if we use std::string instead of sstring. There's an option to alias sstring to std::string, but IIRC it brings a performance regression. If we drop our dependency on strings in the fast path (lookup keyspace, what else?) we could standardize on std::string.